tests/test_dimension.py

```diff
@@ -942,6 +942,27 @@ def test_grouping(self):
         assert len(conds) == 1
         assert len(retrieve_iteration_tree(conds[0].then_body)) == 2
 
+    def test_grouping_v2(self):
+        """
+        Test that two equations sharing a strict subset of loops get scheduled
+        within the same (also shared) ConditionalDimension.
+        """
+        grid = Grid(shape=(10, 10))
+        time = grid.time_dim
+        cond = ConditionalDimension(name='cond', parent=None, condition=time < 5)
```

Comment:
So with the above you can avoid adding None as an arg here?
So we can eventually define COnditional Dimension without specifying parent at all?

Reply:
good catch, fixed